### PR TITLE
Add other godot auto-generated files to gitignore.

### PR DESCRIPTION
**Pull request by Ivorforce**
Referring to how godot-cpp does it: https://github.com/godotengine/godot-cpp/blob/d47758910428242169ebe59329b449edf16036e0/.gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Godot 4+ specific ignores
+# Godot auto generated files
+*.gen.*
+.import/
```
Comment (dsnopek) [inline .gitignore:3]: `.import/` is left over from Godot 3, we don't need that one, and it could be removed from godot-cpp as well

**Review by dsnopek — APPROVED** [review]
I don't think we really _need_ it, because we should already be ignoring any generated files by targeting the `gen/` directories, but it seems fine!

**Comment by dsnopek** [comment]
Er, hang on, I didn't notice this was multiple commits. This'll need to be squashed to a single commit before it can be merged.

See https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase

**Review by paddy-exe — CHANGES_REQUESTED** [review]
@Ivorforce Please squash your commits

**Comment by Ivorforce** [comment]
@paddy-exe done

**Comment by paddy-exe** [comment]
Thanks!

**Pull request merged**